go/enclave/mempool/manager.go

```diff
@@ -95,16 +96,21 @@ func (db *mempoolManager) CurrentTxs(head *obscurocore.Rollup, resolver db.Rollu
 // findTxsNotIncluded - given a list of transactions, it keeps only the ones that were not included in the block
 // todo - inefficient
 func findTxsNotIncluded(head *obscurocore.Rollup, txs []*common.L2Tx, s db.RollupResolver) []*common.L2Tx {
-	included := allIncludedTransactions(head, s)
+	// go back only HeightCommittedBlocks blocks to accumulate transactions to be diffed against the mempool
+	to := uint64(0)
+	if head.NumberU64() > common.HeightCommittedBlocks {
+		to = head.NumberU64() - common.HeightCommittedBlocks
```

Comment:
Would it be clearer to call this `from` or `start` or am I getting confused :) Feels like it's checking transactions from fromHeight to head

Reply:
good point